### ISSUE TITLE
[Dropdown] First Element is Automatically Selected when on Focus

### ISSUE THREAD

**Issue by Lijing29** [microsoft/fluentui#13266]
### Environment Information

- **Package version(s)**: Latest
- **Browser and OS versions**: (fill this out if relevant) Chorme

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/lijinglin29/pen/eYpxZbO

#### Actual behavior:

The first option is automatically selected when on focus

#### Expected behavior:

Provide a method to stop this behavior. It some scenario, we would not like to select an option automatically

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Blocking

Products/sites affected: The product is not GA. If you would like to know, please contact me directly. Thanks.

**Comment by aneeshack4** [comment]
@Lijing29 Thanks for reporting this! I believe this has been reported before/possibly a dupe of #11873 where it was fixed with PR #11876... I don't see any recent changes to Dropdown.base.tsx that would've regressed this. Tagging component owner @joschect - any ideas to what else could cause this?

**Comment by mbest** [comment]
I noticed this as well; when you use the tab key to set focus, the first option is selected. I remember reading somewhere that it's for accessibility.

**Comment by mbest** [comment]
See https://github.com/microsoft/fluentui/issues/10932#issuecomment-545039509

But I think the ARIA guidance is that the first option should be in focus, not necessarily selected.

**Comment by mbest** [comment]
Also #5473

**Comment by DrFlink** [comment]
I worked around this issue by checking the event type. The onChange is triggered on render with event type set to 'focus'. Selecting with mouse or keyboard is triggering "click"-event.

```
onChange={(e, option) => {
    if (e.type === 'focus') return;
    // Do your thing
}}
```

**Comment by xugao** [comment]
based on discussions in #5473, this behavior is correct/by-design:
https://www.w3.org/TR/wai-aria-practices-1.1/#listbox_kbd_interaction

Specifically

> When a single-select listbox receives focus: 
> * If none of the options are selected before the listbox receives focus, the first option receives focus.  Optionally, the first option may be automatically selected.
> * If an option is selected before the listbox receives focus, focus is set on the selected option.
